ui/src/app/subnets/views/SubnetDetails/StaticRoutes/StaticRoutes.tsx

```diff
@@ -164,11 +165,11 @@ const StaticRoutes = ({ subnetId }: Props): JSX.Element | null => {
     <TitledSection
       title="Static routes"
       buttons={
-        isSuperUser ? (
+        isAdmin ? (
           <Button
             disabled={isAddStaticRouteOpen}
             onClick={() => {
-              setIsAddStaticRouteOpen(true);
+              setExpanded({ type: ExpandedType.Create });
```

Comment:
I didn't spot this in review, but it turns out this type already existed in the file.